tests/integration/check-custom-repo/test_custom-repos.py

```diff
@@ -0,0 +1,84 @@
+import re
+import sys
+
+from collections import namedtuple
+
+import pexpect
+import pytest
+
+
+try:
+    from pathlib import Path
+except ImportError:
+    from pathlib2 import Path
+
+from envparse import env
+
+
+def get_system_version(system_release_content=None):
+    """Return a namedtuple with major and minor elements, both of an int type.
+
+    Examples:
+    Oracle Linux Server release 6.10
+    Oracle Linux Server release 7.8
+    CentOS release 6.10 (Final)
+    CentOS Linux release 7.6.1810 (Core)
+    CentOS Linux release 8.1.1911 (Core)
+    """
+    match = re.search(r".+?(\d+)\.(\d+)\D?", system_release_content)
+    if not match:
+        return "not match"
+    version = namedtuple("Version", ["major", "minor"])(int(match.group(1)), int(match.group(2)))
+
+    return version
+
+
+@pytest.mark.good_tests
+def test_good_convertion_with_rhsm(shell, convert2rhel):
+    with convert2rhel(
+        ("-y --no-rpm-va --serverurl {} --username {} --password {} --pool {} --debug").format(
+            env.str("RHSM_SERVER_URL"),
+            env.str("RHSM_USERNAME"),
+            env.str("RHSM_PASSWORD"),
+            env.str("RHSM_POOL"),
+        )
+    ) as c2r:
+        # Make sure subscription-manager installed and attached
+        c2r.expect("The system has been registered with ID")
+
+    assert c2r.exitstatus == 0
+
+
+@pytest.mark.good_tests
+def test_good_convertion_without_rhsm(shell, convert2rhel):
+    """
+    Test if --enablerepo are not skipped when  subscription-manager are disabled and test if repo passed are valid.
+    """
+    with open("/etc/system-release", "r") as file:
+        system_release = file.read()
+        system_version = get_system_version(system_release_content=system_release)
+        if system_version.major == 7:
+            repo = "rhel-7-server-rpms"
+        elif system_version.major == 8:
+            repo = "rhel-8-for-x86_64-baseos-rpms"
```

Comment:
How does one get the repo to the testing VM?

Reply:
checking the `/etc/system-release` file:
```python
    with open("/etc/system-release", "r") as file:
        system_release = file.read()
        system_version = get_system_version(system_release_content=system_release)
        if system_version.major == 7:
            repo = "rhel-7-server-rpms"
        elif system_version.major == 8:
            repo = "rhel-8-for-x86_64-baseos-rpms"
```

Reply:
But the repofile should be specified somewhere. As a secret in .env file? How should the definition look like?

Reply:
After a quick call with @ZhukovGreen, we realize the tests are using our vagrantbox, and we have the repofiles there.

Reply:
I see. Unfortunately the standard vagrant setup described in the [README](https://github.com/oamg/convert2rhel/blob/main/tests/integration/README.md) does not include the repofile definition. One option is making it part of the `.env` file https://github.com/oamg/convert2rhel/blob/main/.env.example. @ZhukovGreen, WDYT?

Reply:
Actually, it does. We are doing vagrant up and our vagrant ansible setup includes role, which adds custom repos. So technically we don't need this step in this MR. However, I just realized that it is going to work just for our local setup. On TFT it won't.

So let's add custom repos, but as a separate MR and into https://github.com/oamg/convert2rhel/tree/main/tests/ansible_collections/roles